apps/desktop/src/index.css

```
@@ -0,0 +1,416 @@
+.dark .text-emerald-400 {
+  color: var(--tokyo-teal) !important;
+}
+
+.dark .bg-emerald-50\/50,
+.dark .bg-emerald-900\/10 {
+  background-color: rgba(115, 218, 202, 0.05) !important;
+}
+
+/* Custom scrollbar */
+.scrollbar-thin {
+  scrollbar-width: thin;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)